.bazelrc

```diff
@@ -19,6 +19,7 @@ build:linux --action_env=SETUPTOOLS_USE_DISTUTILS="false"
 build:linux --action_env=HADOOP_HOME="/tmp/snark/"
 build:linux --action_env=SNARK_LIB_PATH="./src/cc/lib/libwrapper.so"
 build:linux --action_env=LD_LIBRARY_PATH="./external/jvm/jre/lib/amd64/server"
+build:linux --action_env=PYTHONHASHSEED=0
```

Comment:
Please, use env field only for test targets that need it:
https://bazel.build/reference/be/common-definitions#common-attributes-tests